### PR TITLE
implement pipeline 2.0

### PR DESCRIPTION
**Pull request by FeelyChau**

### CODE REVIEW

File: packages/cli/src/bin/pipcook.ts [hunk 1]
```diff
@@ -23,55 +60,26 @@ import serve from '../actions/serve';
...
+    .command('clean')
+    .option('-fm --framework', 'clean cache for framework', true)
+    .option('-s --script', 'clean cache for scripts', true)
+    .action(cacheClean)
+    .description("clean pipcook cache, include framework, script");
 
   program
-    .command('lib')
+    .command('install')
     .description("add libraries: tvm, tensorflow, pytorch...");
```
Comment (yorkie) [inline packages/cli/src/bin/pipcook.ts:83]: The description must be updated :)

File: packages/cli/src/bin/pipcook.ts [hunk 2]
```diff
@@ -23,55 +60,26 @@ import serve from '../actions/serve';
...
-    .command('daemon', 'manage pipcook daemon service')
-    .command('plugin', 'install one or more packages')
-    .command('app', 'experimental PipApp Script')
-    .command('job', 'operate the job bound to specific pipeline')
-    .command('pipeline', 'operate on pipeline');
+    .command('clean')
+    .option('-fm --framework', 'clean cache for framework', true)
+    .option('-s --script', 'clean cache for scripts', true)
+    .action(cacheClean)
+    .description("clean pipcook cache, include framework, script");
```
Comment (yorkie) [inline packages/cli/src/bin/pipcook.ts:79]:
```suggestion
    .description('clean pipcook cache, include framework and script');
```

File: packages/cli/src/bin/pipcook.ts [hunk 3]
```diff
@@ -23,55 +60,26 @@ import serve from '../actions/serve';
...
+    .action(run);
 
   program
-    .command('daemon', 'manage pipcook daemon service')
-    .command('plugin', 'install one or more packages')
-    .command('app', 'experimental PipApp Script')
-    .command('job', 'operate the job bound to specific pipeline')
-    .command('pipeline', 'operate on pipeline');
+    .command('clean')
+    .option('-fm --framework', 'clean cache for framework', true)
```
Comment (yorkie) [inline packages/cli/src/bin/pipcook.ts:76]:
```suggestion
    .option('-f --framework', 'clean cache for framework', true)
```

File: packages/cli/src/runtime.ts [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+import { PipelineMeta } from '@pipcook/pipcook-core';
+import { Costa } from '@pipcook/costa';
+import * as path from 'path';
+import { createStandaloneRT } from './standalone-impl';
+import { logger, Framework, Plugin, Script } from './utils';
+
+/**
+ * runtime for local
+ */
+export class StandaloneRuntime {
```
Comment (yorkie) [inline packages/cli/src/runtime.ts:11]: Write more comments for this class.

File: packages/cli/src/standalone-impl.ts [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+  Sample,
+  Runtime,
+  TaskType,
+  DefaultType,
+  ProgressInfo,
+  pipelineAsync
+} from '@pipcook/pipcook-core';
+import * as fs from 'fs-extra';
+import * as path from 'path';
+export class DataAccessorImpl<T> implements DataAccessor<T> {
```
Comment (yorkie) [inline packages/cli/src/standalone-impl.ts:16]: Add a new line.

File: packages/cli/src/utils/cache.test.ts [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+  await fetchWithCache(cacheDir, url, target);
+
+  t.false(stubDownloadAndExtractTo.called);
+  t.true(stubRemove.calledOnce);
+  t.true(stubPathExists.calledOnce);
+  t.true(stubSymlink.called);
+});
+
+test.serial('fetch with missed cache', async (t) => {
+  const cacheDir = ".cache";
```
Comment (yorkie) [inline packages/cli/src/utils/cache.test.ts:35]:
```suggestion
  const cacheDir = '.cache';
```

File: packages/cli/src/utils/cache.test.ts [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+  const url = "url";
```
Comment (yorkie) [inline packages/cli/src/utils/cache.test.ts:36]: Ditto.

File: packages/cli/src/utils/cache.test.ts [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+  const target = "target";
```
Comment (yorkie) [inline packages/cli/src/utils/cache.test.ts:37]: Ditto.

**Review by yorkie — CHANGES_REQUESTED** [review]
@FeelyChau May I ask you to check the warnings and formats?

**Review by yorkie — APPROVED** [review]

**Review by WenheLI — APPROVED** [review]
LGTM

**Review by rickycao-qy — APPROVED** [review]
LGTM

**Pull request merged**